docs/API.md

```diff
@@ -1353,6 +1360,22 @@ type StatsCount struct {
 
 If the refresh boolean parameter is non-zero, the latest stats will be returned without caching.
 
+#### GET *id*/meta/supported-series
+
+The `supported-series` path returns the OS series that are defined by the entity as
```

Comment:
I've renamed to charm-supported-series and added an extra example to show how an empty list is returned if no supported series are declared in the metadata.